atmo/clusters/models.py

```diff
@@ -94,16 +102,18 @@ def save(self, *args, **kwargs):
 
         # set the dates
         if not self.start_date:
-            self.start_date = datetime.now().replace(tzinfo=UTC)
+            self.start_date = timezone.now()
```

Comment:
Since `timezone.now()` will have microsecond accuracy it's a minor nit to call it again to set `end_date`. Instead set it into a separate variable and add the timedelta to that for `end_date`